README.md

```diff
@@ -45,21 +46,63 @@ This is the dependency you need:
 </dependency>
 ```
 
-Read the [Javadoc](http://www.javadoc.io/doc/com.artipie/asto)
-for more technical details.
+Read the [Javadoc](http://www.javadoc.io/doc/com.artipie/asto) for more technical details. If you 
+have any question or suggestions, do not hesitate to create as issue or contact us in 
+[Telegram](https://t.me/artipie), our roadmap is available [here](https://github.com/orgs/artipie/projects/3/views/1).
 
 # Usage
 
-The main types here are:
+The main entities here are:
  - `Storage` interface provides API for key-value storage
  - `Key` represents storage key
  - `Content` represents storage binary value
 
-[Storage](https://www.javadoc.io/doc/com.artipie/asto/latest/com/artipie/asto/Storage.html)
-[Key](https://www.javadoc.io/doc/com.artipie/asto/latest/com/artipie/asto/Key.html) and other types are
+[Storage](https://www.javadoc.io/doc/com.artipie/asto/latest/com/artipie/asto/Storage.html),
+[Key](https://www.javadoc.io/doc/com.artipie/asto/latest/com/artipie/asto/Key.html) and other entities are
 documented in [javadoc](https://www.javadoc.io/doc/com.artipie/asto/latest/index.html).
 
-TO DO: add more details
+Here is en example of how to create `FileStorage`, save and then read some data:
+```java
+final Storage asto = new FileStorage(Path.of("/usr/local/example"));
+final Key key = new Key.From("hello.txt");
+asto.save(
+    key,
+    new Content.From("Hello world!".getBytes(StandardCharsets.UTF_8))
+).thenCompose(
+    ignored -> asto.value(key)
+).thenCompose(
+    val -> new PublisherAs(val).asciiString()
+).thenAccept(
+    System.out::println
+).join();
+```
+In the example we created local text file `/usr/local/example/hello.txt` containing string "Hello world!",
+then read and print it into console. Used classes:
+- `Key.From` is implementation of the `Key` interface, keys are strings, separated by `/`
+- `Content.From` implements `Content` interface, allows to create `Content` instances from 
+  byte arrays or publisher
```

Comment:
publisher -> publisher of ByteBuffer's.
And link for publisher: https://www.reactive-streams.org/reactive-streams-1.0.4-javadoc/org/reactivestreams/Publisher.html.

What do you think?